src/frontend/src/eth/components/send/ConvertToCkETH.svelte

```diff
@@ -28,7 +28,7 @@
 	ariaLabel={$i18n.convert.text.convert_to_cketh}
 >
 	<IconConvert size="28" slot="icon" />
-	<span> {$i18n.convert.text.convert_to_cketh} </span>
+	<span>{$ethereumToken.twinTokenSymbol}</span>
```

Comment:
in that specific store case, yes

Reply:
Sometimes I'm paranoid, just thinking at this component moving in the future and having someday `undefined` printed as text in the UI. Would you mind defaulting to empty string `?? ""` just in case sh** happens someday?

Reply:
Done!

There is actually an ES lint rule that detects unnecessary coalescing checks, that we could use to define the types better (check which one should be `| undefined`)